README.md

```diff
@@ -80,7 +80,7 @@ Investigations into [protecting openers](https://groups.google.com/a/chromium.or
 
 The following would be added to [windowFeatures](https://developer.mozilla.org/en-US/docs/Web/API/Window/open#windowfeatures):
 
-* popin - Indicates the window should be opened as an inline popin. The target must be `_blank` as others have no special meaning in this context. Cannot be used with `popup` feature as these options conflict. The features `width`, `height`, `left`, and `top` may or may not be respected by the rendering browser (especially on mobile). `noopener` and `noreferrer` will be respected as usual.
+* popin - Indicates the window should be opened as an inline popin. The target must be `_blank` as others have no special meaning in this context. Cannot be used with `popup` feature as these options conflict. The features `width`, `height`, `left`, and `top` may or may not be respected by the rendering browser (especially on mobile). It will not be possible to use `noopener` or `noreferrer` (attempting to do so will cause rejection of popin creation) as the popin must know the referrer to gain access to the correct storage partition and the blocking nature of popins negates any utility in having unreachable popins (a window would want to know when they were closed).
```

Comment:
> and the blocking nature of popins negates any utility in having unreachable popins

This bit is hard to parse. Could you help me understand what it's trying to say?

Reply:
(1) noopener is a bad idea because of the properties of Popins. Specifically, they block access to the opening window until closed and the opened window has some knowledge of who opened them as they can require only specific origins do so
(2) noreferrer might be a lost cause? It's true that info is restricted, but at least some knowledge about the origin of the referrer is known

@johannhof's point is that we can't just casually violate the Referrer-Policy and in some cases noreferrer seems to imply nooepner

Perhaps we just have to allow the interaction to be weird, and allow them as-is